db/tf2idb.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env python
-
-TF_FOLDER = 'C:/Program Files (x86)/Steam/steamapps/common/Team Fortress 2/tf/'
-ITEMS_GAME = TF_FOLDER + 'scripts/items/items_game.txt'
-DB_FILE = 'tf2idb.sq3'
+WINDOWS_DEFAULT_TF_FOLDER = 'C:/Program Files (x86)/Steam/steamapps/common/Team Fortress 2/tf/'
+WINDOWS_DEFAULT_MANIFEST = WINDOWS_DEFAULT_TF_FOLDER + 'scripts/items/items_game.txt'
```

Comment:
We should probably add a slash at the beginning (double slashes are always collapsed to a single one) in case someone changes ``WINDOWS_DEFAULT_TF_FOLDER`` to something that does not end in a slash.